README.md

```diff
@@ -40,6 +40,9 @@ import PlaytBrowserClient from '@playt/client/browser';
 const browserClient = PlaytBrowserClient({
   apiUrl: '<API_URL>',
 });
+await browserClient.initialize({
```

Comment:
the change is still technically not breaking, because not calling the initialize doesn't really break any of the existing functionality